packages/client/hooks/useAnimatedSpotlightSource.ts

```diff
@@ -43,11 +43,15 @@ const useAnimatedSpotlightSource = (
     }, 0)
     dragIdRef.current = clientTempId()
     // execute mutation after cloning as the mutation will cause reflection height to change
-    StartDraggingReflectionMutation(atmosphere, {
-      reflectionId,
-      dragId: dragIdRef.current,
-      isSpotlight: true
-    })
+    const dragInterval = setInterval(() => {
+      // execute every second so that the remote animation continues when subscribers refresh page
```

Comment:
To test: 
- remove interval
- client A opens Spotlight so client B sees remote animation 
- client B refreshes the page & the remote animation stops